packages/lesswrong/components/posts/SubmitToFrontpageCheckbox.tsx

```diff
@@ -49,13 +48,15 @@ const styles = (theme: ThemeType): JssStyles => ({
     }
   },
   checkboxLabel: {
-    fontWeight:500,
     fontFamily: theme.typography.commentStyle.fontFamily,
-    fontSize: 16,
-    color: theme.palette.text.dim40,
+    fontSize: 14,
+    color: theme.palette.grey[680],
     verticalAlign: 'middle',
     lineHeight: '1.25em'
   },
+  checkbox: {
+    padding: 8
```

Comment:
```suggestion
    padding: 6
```

Reply:
Seeing this live, I think 6px radius would be better than the 8px I specified in the designs 👍